.taskcluster.yml

```diff
@@ -224,6 +224,42 @@ tasks:
             owner: ${user}@users.noreply.github.com
             source: ${repository}/raw/${head_rev}/.taskcluster.yml
 
+      - $if: 'tasks_for == "github-push" && head_branch[:10] == "refs/tags/"'
```

Comment:
We should deploy at the end of the data pipeline

Reply:
If we do that, that means we need to wait for a full data-pipeline in order to deploy any code change, including a hotfix, that's sub-obtimal. But we are generating an HTTP Service Docker image without the models so it's effectively broken.

I'm not fan of either solutions, @marco-c what do you think?

Reply:
For now we have to do the non-broken thing, we can figure out how to deploy fixes later (file an issue so we don't forget).

Reply:
Done https://github.com/mozilla/bugbug/issues/486